src/Swan.Lite/Extensions.Enumerable.cs

```diff
@@ -4,19 +4,29 @@
 namespace Swan
 {
     /// <summary>
-    /// This class contains extensions methods for types implementing IEnumerable<T>
+    /// This class contains extension methods for types implementing IEnumerable&lt;TSource&gt;
     /// </summary>
     public static class EnumerableExtensions
     {
         /// <summary>
-        /// This method returns the <see cref="System.Linq.Enumerable.Union"/>
+        /// This method returns the <see cref="Enumerable.Union{TSource}">Union</see>
         /// of all non-null parameters.
         /// </summary>
-        /// <returns> An <see cref="IEnumerable{T}"/> that contains the elements from non-null input sequences, excluding duplicates. </returns>
-        public static IEnumerable<T> UnionExcludingNulls<T>(this IEnumerable<T> @this, IEnumerable<T> second)
-        {
-            if (@this == null) return second ?? Enumerable.Empty<T>();
-            return second == null ? @this : @this.Union(second);
-        }
+        /// <returns> An <see cref="IEnumerable{TSource}"/> that contains the elements from non-null input sequences, excluding duplicates. </returns>
```

Comment:
Please document parameters. The documentation for `Enumerable.Union` can be a good starting point.